scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java

```diff
@@ -424,7 +431,13 @@ public void restartTaskOnNodeFailure(final InternalTask task) {
             @Override
             public void run() {
                 TerminationData terminationData = jobs.restartTaskOnNodeFailure(task);
-                terminationData.handleTermination(SchedulingService.this);
+                try {
+                    terminationData.handleTermination(SchedulingService.this);
+                } catch (IOException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1166)